### PR TITLE
Issue_524:Output parameter from generate_report function can be removed

### PR DESCRIPTION
**Pull request by geetanjalimanegslab**
# Please include a summary of the changes

* Updated  vane/tests_tools.py with  generate_report utils 

# Any specific logic/part of code you need extra attention on
  Updated   vane/tests_tools.py   generate_report utils function,  set the default value for output argument so that older test cases won't get impacted with this change, and for new test cases, we will not pass the output as the argument.

# Include the Issue number and link

Make sure to link the issue in the github PR UI 
https://github.com/aristanetworks/vane/issues/524

# List/Attach any dependencies/past issues that are required for this change/provide context

# Type of change

- - [ ] Bug fix (non-breaking change which fixes an issue)
- - [ ] New feature (non-breaking change which adds functionality)
- - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- - [ ] This change requires a documentation update
- - [ ] Other (please specify)

# Effort required on reviewers end

- - [x ] Easy
- - [ ] Medium
- - [ ] Hard 

# How Has This Been Tested?

## Bug fix

    Include before and after snapshots/screenshots/changed logs
    
## New feature

    Ensure current test cases pass and include newer test cases if required
    
# CI pipeline result

- - [ ] Pass
- - [ ] Fail
  
  If it fails, explain the reason and whether or not we should ignore the failure
  
# Verify Documentation Update

If applicable, ensure changes to [official documentation](http://vane.arista.com/) are included in the PR by following the steps below:

(1) You can locally make required documentation changes within the [docs](https://github.com/aristanetworks/vane/tree/develop/docs) folder and test them by following these steps:

- `mike deploy test`(or any name you want for the local changes version)
- `mike serve`
  (you will be able to view `test` as an option in the version dropdown)
- Once you have ensured the changes are accurate, execute `mike delete test`(this ensures you do not have any local  version branches which do not exist on gh-pages in the github repo).  

(2) Ensure you have [pre-commit hook](https://pre-commit.com/) installed which will take care of linting checks for documentation before you commit your changes.

(3) Lastly ensure the following jobs pass when you create the PR. These jobs ensure that the changes within your PR do not break the documentation.

- Documentation Testing / Run pre-commit validation hooks (pull_request) 
- Documentation Testing / Build site with no warnings (pull_request)  
   

    
# Additional comments

### CODE REVIEW

File: vane/tests_tools.py [hunk 1]
```diff
@@ -1218,11 +1218,13 @@ def _get_parameters(self, tests_parameters, test_suite, test_case):
...
 
-    def generate_report(self, dut_name, output):
-        """Utility to generate report
-
+    def generate_report(self, dut_name, output=""):
+        """
+        Utility to generate an HTML/DOCX report
         Args:
           dut_name (str): name of the device
+          output(str): Output of commands ran during the test execution.
```
Comment (vishwajeetgslab) [inline vane/tests_tools.py:1226]:
```suggestion
          output (str): Output of commands executed on device. Default value is empty string.
```

**Review by vishwajeetgslab — CHANGES_REQUESTED** [review]

**Review by vishwajeetgslab — APPROVED** [review]

**Comment by kgrozis-arista** [comment]
@vishwajeetgslab Documentation pipeline is failing with this commit.  Can you look into this?

**Comment by vishwajeetgslab** [comment]
> @vishwajeetgslab Documentation pipeline is failing with this commit. Can you look into this?

Hi @kgrozis-arista , Documentation pipeline is fixed now. Thanks

**Review by kgrozis-arista — APPROVED** [review]
LGTM

**Pull request merged**